.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/mus/Projects/httpx/venv/bin/python3"
+}
```

Comment:
We don't really want this file. You might want to setup a global gitignore to avoid including config that's specific to your local dev environment.

Reply:
Slipped me. Fixing.